src/components/templates/Auditorium/components/Section/Section.tsx

```diff
@@ -18,12 +22,21 @@ import "./Section.scss";
 export const SECTION_SEAT_SIZE = "var(--section-seat-size)";
 export const SECTION_SEAT_SIZE_MIN = "var(--section-seat-size-min)";
 export const SECTION_SEAT_SPACING = "var(--section-seat-spacing)";
+export const REACTIONS_CONTAINER_HEIGHT_IN_SEATS = 2;
 
 export interface SectionProps {
   venue: WithId<AuditoriumVenue>;
 }
 
 export const Section: React.FC<SectionProps> = ({ venue }) => {
+  const { userWithId } = useUser();
+  const [isAudioEffectDisabled, setIsAudioEffectDisabled] = useState(false);
+  // @debt refactor via useShowHide hook
+  const toggleMute = useCallback(
+    () => setIsAudioEffectDisabled((state) => !state),
+    []
+  );
```

Comment:
useShowHide. Why refactor if it can be done in 3 mins?

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1339/commits/b7df1b93085778a251ef67086930372cbe4adc98